code/qcommon/q_platform.h

```diff
@@ -189,14 +189,8 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 #define PATH_SEP '/'
 
 #if !defined(ARCH_STRING)
-#if INTPTR_MAX == INT64_MAX
-#define ARCH_STRING "x86_64"
-#elif INTPTR_MAX == INT32_MAX
-#define ARCH_STRING "x86"
-#else
```

Comment:
NOTE: Adopt upstream ioq3 engine's `code/qcommon/q_platform.h` and let `ARCH_STRING` to be defined by `CMakeLists.txt`